deploy-edge.js

```diff
@@ -0,0 +1,170 @@
+import url from 'url';
+import { EOL } from 'os';
+import AWS from 'aws-sdk';
+import axios from 'axios';
+import JSZip from 'jszip';
+import { promisify } from 'util';
+import { promises as fs } from 'fs';
+import parseEnvString from 'parse-env-string';
+
+const { Lambda, CloudFront, Credentials } = AWS;
+
+const REGION = 'us-east-1';
+
+export default {
+  command: 'edge',
+  describe: 'Deploy an AWS Lambda@Edge function used by a CloudFront distribution',
+  builder: yargs => yargs
+    .option('access-key', { demandOption: true, type: 'string' })
+    .option('secret-key', { demandOption: true, type: 'string' })
+    .option('lambda-env', { type: 'array', string: true })
+    .option('lambda-env-inject', { demandOption: false, type: 'string' })
```

Comment:
Tiny bit of inconsistency here with specifying `demandOption: false` or omitting it altogether.

Reply:
True, I removed it here also